Formula/g/gh-ost.rb

```diff
@@ -0,0 +1,23 @@
+class GhOst < Formula
+  desc "Triggerless online schema migration solution for MySQL"
+  homepage "https://github.com/github/gh-ost"
+  url "https://github.com/github/gh-ost/archive/refs/tags/v1.1.7.tar.gz"
+  sha256 "f3ebf966a84ddeb97e5dd0a70dd027c2d088c71dd18f72c1ac75373f2be430ab"
+  license "MIT"
+  head "https://github.com/github/gh-ost.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X main.AppVersion=#{version} -X main.GitCommit=#{tap.user}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./go/cmd/gh-ost"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/gh-ost -version")
```

Comment:
This is working but we may use `--version` for consistency.

Reply:
it is actually from `--help`

```
$ gh-ost --help
Usage of gh-ost:
...
  -verbose
        verbose
  -version
        Print version & exit
```